presto-hive/src/main/java/com/facebook/presto/hive/StoragePartitionLoader.java

```diff
@@ -261,11 +262,13 @@ public StoragePartitionLoader(
             return addSplitsToSource(splits, splitFactory, hiveSplitSource, stopped);
         }
         PathFilter pathFilter = isHudiParquetInputFormat(inputFormat) ? hoodiePathFilterLoadingCache.getUnchecked(configuration) : path1 -> true;
+        // Streaming aggregation works at the granularity of individual files
         // S3 Select pushdown works at the granularity of individual S3 objects,
         // Partial aggregation pushdown works at the granularity of individual files
         // therefore we must not split files when either is enabled.
         // Skip header / footer lines are not splittable except for a special case when skip.header.line.count=1
         boolean splittable = isFileSplittable(session) &&
+                !isStreamingAggregationEnabled(session) &&
```

Comment:
hmmm if we are already guarding `splittable` here through `isStreamingAggregationEnabled`, do we still need to introduce `isFileSplittable` config at all? cc: @yuanzhanhku @arunthirupathi

Reply:
isFileSplittable is not needed for streaming aggregation support. Ke just added it for benchmarking propose. It allows us to run a query with file spilling disabled and streaming aggregation enable/disabled so that we can measure the improvement of steaming aggregation without the impact from file splitting.